### ISSUE TITLE
[Bug]: Lock session when AnyDesk

### ISSUE THREAD

**Issue by LukasCCB** [beyondcode/herd-community#768]
### Platform

Windows

### Operating system version

Windows 11 23H2 

### System architecture

Windows

### Herd Version

Last (I can't see)

### Bug description

When AnyDesk (Administrador permissions) session. Open Herd and lock all session usabled.

This exact problem occurred when running the Task Manager, and then when Full Administrator permission is given to the AnyDesk visitor, it makes this action possible.

But the same does not happen at Herd.

### Steps to reproduce

1. Connect AnyDesk session.
2. Host mark "Mark Administrator Access" to YES.
3. Client session open Herd app.
4. All session is locked, need ask to Host close and open Herd again.

**Comment by sschlein** [comment]
Please let me rephrase the issue so that I make sure I understand the problem correctly:

You use AnyDesk from a local machine to connect to a remote machine. The remote machine runs Herd and you work on this machine, but this doesn't work as expected and has the problems that you describe above?
